### DISCUSSION TITLE
Delete all shops of a specific player command

### DISCUSSION THREAD

**Discussion by MrPranklez** [QuickShop-Community/QuickShop-Hikari#857] [Ideas]
This is something I recently encountered and it was rather tedious to solve manually. I had to delete all stores created by a player. Locating all these store proved to be a challenge since I had to go digging in the db files. So first of all it might be interesting to have a `/qs list <playerName>` command to just list the shops that someone has with some coordinates or a clickable message to teleport for instance.

And then the main reason why I created this discussion was a command that would simply delete all the shops that 1 player had created. Something like `/qs removeall <playerName>`. I found the `/qs removeworld` command but that would remove all the shops in a world instead of all the shops of a specific individual.

**Comment by Ghost-chu** [comment]
The feature you requesting seems already exists.

https://ghost-chu.github.io/QuickShop-Hikari-Documents/docs/modules/shops/remove-shops#remove-someones-all-shops
